tool/generate_font.dart

```diff
@@ -117,6 +118,7 @@ void main(List<String> arguments) {
     "import 'package:font_awesome_flutter/src/icon_data.dart';",
     "export 'package:font_awesome_flutter/src/fa_icon.dart';",
     "export 'package:font_awesome_flutter/src/icon_data.dart';",
+    "export 'package:font_awesome_flutter/src/fa_duotone_icon.dart';",
```

Comment:
This line should only be added if there are duotone icons (condition in lines [122-125](https://github.com/fluttercommunity/font_awesome_flutter/blob/master/tool/generate_font.dart#L122-L125)). I'd like to keep it that way as most of the users of this package will not have pro and thus not access to duotone icons. It keeps the api much clearer

Reply:
This is actually required for the example app to work - if I remove it [line 88 of `example/lib/main.dart`](https://github.com/amkuchta/font_awesome_flutter/blob/duotone-updates/example/lib/main.dart#L88) throws an error. Since it's an export, it shouldn't affect the file size - it's already included in the code that is generated, just not presented for use (the readme file doesn't have any instruction on how to make FaDuotoneIcon usable - it just says that it has to be used. This exposes it for those who need it without impacting the size of the package).